addons/mail/static/src/models/channel.js

```diff
@@ -148,6 +148,18 @@ registerModel({
             }
             return this.isDescriptionEditable;
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsRenamable() {
+            const typesOfRenamableChannel = new Set([
+                'channel',
+                'chat',
+                'group',
+            ]);
```

Comment:
Already said in another comment:  no gain with `Set`, actually worse performance because creation of the `Set` is quite costly in theory

Reply:
changes done